### ISSUE TITLE
Error 500 when trying to view custom event properties

### ISSUE THREAD

**Issue by Jaarson** [umami-software/umami#2901]
### Describe the Bug

After upgrading from v2.12.1 to v2.13.0 I can't view events custom data (properties)

Endpoint `/api/websites/<website-id>/event-data/values` responds with error code 500

### Database

PostgreSQL

### Relevant log output

```shell
PrismaClientKnownRequestError: 
Invalid `prisma.$queryRawUnsafe()` invocation:


Raw query failed. Code: `42804`. Message: `ERROR: CASE types character varying and timestamp with time zone cannot be matched`
    at _n.handleRequestError (/app/node_modules/@prisma/client/runtime/library.js:121:7749)
    at _n.handleAndLogRequestError (/app/node_modules/@prisma/client/runtime/library.js:121:7057)
    at _n.request (/app/node_modules/@prisma/client/runtime/library.js:121:6741)
    at async l (/app/node_modules/@prisma/client/runtime/library.js:130:9355)
    at async l (/app/.next/server/pages/api/websites/[websiteId]/event-data/values.js:1:2091)
    at async K (/app/node_modules/next/dist/compiled/next-server/pages-api.runtime.prod.js:20:16853)
    at async U.render (/app/node_modules/next/dist/compiled/next-server/pages-api.runtime.prod.js:20:17492)
    at async NextNodeServer.runApi (/app/node_modules/next/dist/server/next-server.js:600:9)
    at async NextNodeServer.handleCatchallRenderRequest (/app/node_modules/next/dist/server/next-server.js:269:37)
    at async NextNodeServer.handleRequestImpl (/app/node_modules/next/dist/server/base-server.js:816:17) {
  code: 'P2010',
  clientVersion: '5.17.0',
  meta: {
    code: '42804',
    message: 'ERROR: CASE types character varying and timestamp with time zone cannot be matched'
  }
}
```
```


### Which Umami version are you using? (if relevant)

v2.13.0

### Which browser are you using? (if relevant)

_No response_

### How are you deploying your application? (if relevant)

Docker

**Comment by franciscao633** [comment]
Thanks for reporting. We get get this fixed asap.

**Comment by mikecao** [comment]
Fixed in v2.13.1

**Comment by Jaarson** [comment]
Confirming it now works after updating to v2.13.1, thank you!
